docs/config.md

```diff
@@ -601,7 +600,7 @@ Examples:
 
 #### `Gateway.PublicGateways: Paths`
 
-Array of paths that should be exposed on the hostname.
+An array of paths that are exposed on the hostname.
```

Comment:
I would stick with "should be". That is, you're telling IPFS what to do, not what "is".

Reply:
That or "An array of paths that ipfs should expose".